pf4j/src/main/java/ro/fortsoft/pf4j/DefaultPluginManager.java

```diff
@@ -131,6 +131,23 @@ public DefaultPluginManager(File pluginsDirectory) {
         initialize();
     }
 
+    /**
+     * Returns the currently used direcotry containing the plugins
+     * @return File pointing to the plugins directory
+     */
+    public File getPluginsDirectory() {
+        return pluginsDirectory;
+    }
+
+    /**
+     * Returns the PluginClasspath used by this plugin manager
+     * @return  the classpath used by this manager
+     */
+    protected PluginClasspath getPluginClasspath() {
```

Comment:
I don't think that we need this method. From your gist code I see that you use this method in 
```java
protected PluginDescriptorFinder createPluginDescriptorFinder() {
    if (RuntimeMode.DEVELOPMENT.equals(getRuntimeMode())) {
        return new PropertiesPluginDescriptorFinder();
    }

    return new DefaultPluginDescriptorFinder(getPluginClasspath());
}
```
but your code is exactly the same with the default implementation:
```java
protected PluginDescriptorFinder createPluginDescriptorFinder() {
    if (RuntimeMode.DEVELOPMENT.equals(getRuntimeMode())) {
        return new PropertiesPluginDescriptorFinder();
    }

    return new DefaultPluginDescriptorFinder(pluginClasspath);
}
```